platform-emf/src/main/java/com/softicar/platform/emf/management/importing/variable/find/EmfImportVariablesFinder.java

```diff
@@ -6,7 +6,7 @@
 
 public class EmfImportVariablesFinder {
 
-	private static final String VARIABLE_PATTERN = "\\$\\$[a-z]\\$\\$";
+	private static final String VARIABLE_PATTERN = "\\$\\$[a-z]+\\$\\$";
```

Comment:
i think we should also support underscore and uppercase letters as well as numbers

Reply:
See new ACCs of story.